switching/input/messages/Deadlines.py

```diff
@@ -1,10 +1,10 @@
 from collections import namedtuple
-from datetime import timedelta
+from datetime import datetime, timedelta
 from workdays import workday
 
 
 class DeadLine(namedtuple('Deadline', "step, days, next_step")):
-    def limit(self, date):
+    def limit(self, date=datetime.today()):
```

Comment:
It's better to initialize with `None` and then check `if date is None` assign `datetime.today()`